Yao-2017-1100 affective norms for Chinese

AnnikaTjuka opened the pull request:
# Pull request checklist

- [X] add new concept list
- [ ] add new metadata
- [ ] add new Concepticon concept sets
- [ ] add new Concepticon concept relations
- [ ] refine existing Concepticon concept set mappings
- [ ] refine Concepticon glosses
- [ ] refine Concepticon concept relations
- [ ] refine Concepticon concept definitions
- [ ] retire data

## Additional information

About the data:
- This list contains adjectives, nouns and verbs.
- I corrected some obvious spelling mistakes in the English translations.

About the mapping:
- A lot of the English translations occur two or three times. 
- I tried to use each CONCEPTICON_GLOSS only once. Except of ENEMY, because we have two Chinese glosses already mapped.

@MacyL Because I mainly relied on the English translations, you might spot some additional concepts that could be mapped.

LinguList commented:
@AnnikaTjuka, I just re-assigned the reviewers. I will this time make the role of the manager of the PR, so it is my obligation to accept this PR, and to make sure that all changes by the reviewers are handled. I assigned @Kristina-Pianykh, @MacyL, and @CarolinHu. I did not assign @Schweikhard since he's on sick leave. We will wait until we have review reports by all three of them, and I'll check with the reports and moderate this. 

@Kristina-Pianykh and @CarolinHu, if you feel lost or are not sure, you can also just mention that.

In the future, everybody who submits a concept list should first determine the moderator and not assign reviewers directly. The moderator will decide this, and all concepticon editors can be moderator.

I hope  in this way we can start to organize the mapping procedure etc. more properly.

MacyL commented:
I haven't looked into the list carefully, but I spotted these 7 items can already be mapped to concepticon.  The others are either very specific, like 荒島 and 孤島, or synonyms  like 欢乐 and 欢喜
I will need some time to go through them. 
| source               | CONCEPTICON CONCEPT | CONCEPTICON IDs |
|----------------------|---------------------|-----------------|
| marry out            | MARRY (AS WOMAN)    | 2163            |
| young lady           | GIRL                | 1646            |
| nurse                | NURSE               | 1104            |
| wreath for the death | WREATH              | 2589            |
| farewell             | GOODBYE             | 3554            |
| desert               | DESERT              | 3687            |
| home coming          | RETURN HOME         | 142             |

CarolinHu commented on concepticondata/conceptlists/Yao-2017-1100.tsv:
```diff
@@ -0,0 +1,1101 @@
+Yao-2017-1100-757	757	烧伤	burn	BURN	2102	2.63	7.03	4.83	5.5	4.79	5.23
+Yao-2017-1100-758	758	稍微	a bit	A LITTLE	2924	2.54	5.44	5.37	3.97	6.1	5.25
+Yao-2017-1100-759	759	少女	young lady	GIRL	1646	6.57	5.37	6.93	6.8	6.64	6.17
+Yao-2017-1100-760	760	申请	apply	APPLY (OINTMENT)	3650	5.79	3.18	6.75	6.82	7.1	6.18
```
If the dictionaries I found are any good, this seems to be more like 'apply for (e.g. a job)'

LinguList replied:
Yes, this means exactly "sich bewerben" or similar.

CarolinHu commented on concepticondata/conceptlists/Yao-2017-1100.tsv:
```diff
@@ -0,0 +1,1101 @@
+Yao-2017-1100-868	868	死囚	Death row inmates			2.29	6.85	3.78	3.68	5.77	4.97
+Yao-2017-1100-869	869	死刑	death penalty			4.64	4.29	6.5	3.07	6.48	5.54
+Yao-2017-1100-870	870	死罪	death penalty			5.1	4.95	4.88	5.27	6.62	6.32
+Yao-2017-1100-871	871	寺院	temple	TEMPLES	1604	5.05	5.09	6.09	3.95	6.71	5.66
```
Google pictures claims this is 911 TEMPLE.

LinguList replied:
Oh yes, important spot, @CarolinHu, this is a Tempel.

LinguList commented on concepticondata/conceptlists/Yao-2017-1100.tsv:
```diff
@@ -0,0 +1,1101 @@
+Yao-2017-1100-61	61	病态	sick	BECOME SICK	2327	6.2	5.84	5.37	5.37	6.75	6.08
+Yao-2017-1100-62	62	波浪	waves	WAVE	978	5.36	5.21	3.32	5.43	4.96	5.69
+Yao-2017-1100-63	63	搏斗	fight	BRAWL (FIGHT)	2902	1.77	6.73	3.13	6.33	3.61	4.25
+Yao-2017-1100-64	64	捕杀	killing			1.87	6.9	3.53	3.13	4.96	4.97
```
It is correct to not map it, since this means in German something like "Gemetzel", or "morden", so it is stronger than "to kill".

LinguList commented on concepticondata/conceptlists/Yao-2017-1100.tsv:
```diff
@@ -0,0 +1,1101 @@
+Yao-2017-1100-295	295	果盘	fruit tray			3.88	4.57	4.37	5.83	4.87	5.42
+Yao-2017-1100-296	296	果树	fruit tree			4.8	3.14	6.14	5.58	6.23	5.79
+Yao-2017-1100-297	297	过程	process			4.78	4.61	3.08	5.67	3.6	4.28
+Yao-2017-1100-298	298	海岸	coastline			2.37	6.1	4.95	5.18	4.39	4.97
```
This is in German the "Ufer des Meeres". @AnnikaTjuka, I let you decide if this fits with "shore of sea".

MacyL commented on concepticondata/conceptlists/Yao-2017-1100.tsv:
```diff
@@ -0,0 +1,1101 @@
+Yao-2017-1100-201	201	风铃	wind pipe			5.36	5.27	5.55	4.77	6.29	5.76
+Yao-2017-1100-202	202	风沙	sand storms			2.29	6.93	3.75	3.64	5.73	5.63
+Yao-2017-1100-203	203	风扇	electric fan			7.13	5.1	7.03	6.33	7.54	6.42
+Yao-2017-1100-204	204	风俗	ritual			4.8	5.09	5.72	5.92	5.59	5.89
```
风俗 CUSTOM [1747] would be a better fit.

MacyL commented:
疤痕 SCAR [497]

LinguList commented:
@AnnikaTjuka, if you replied to the cases mentioned there (there are just a few), can you notify me, so I can "moderate" this? And @MacyL should then approve.

AnnikaTjuka commented:
Thanks for your comments! I discussed some ambiguous cases directly with @MacyL . For the sake of completeness, here is the result of our discussion:

Chinese | English | concept | ID
-- | -- | -- | --
出嫁 | marry out | MARRY (AS WOMAN) | 2163
姑娘 | young lady | YOUNG WOMAN | 77
少女 | young lady | GIRL | 1646
沙漠 | desert | DESERT | 3687
风俗 | ritual | CUSTOM | 1747
礼仪 | ritual | RITE | 3334
伤疤 | scar | SCAR | 497

I also changed the mappings according to the comments by @CarolinHu and added the ones @MacyL  proposed. I decided to not map coastline.

LinguList commented:
Fine by me, @MacyL, if you checked this, can you submit your review as "approve"?

LinguList commented:
Okay, I think we have reviewed enough, so we can ask @Kristina-Pianykh to help with other lists. Please merge, if you are ready, @AnnikaTjuka.
